content/en/install.md

```diff
@@ -3,101 +3,149 @@ title: Installation
+see [Installing with type stubs](#type-stubs).
 
-<a name="distributions"></a>
+[static type stubs]: https://typing.readthedocs.io/en/latest/guides/libraries.html
 
-# Scientific Python Distributions (recommended)
+<a name="environment-based"></a>
 
-Python distributions provide the language itself, along with the most
-commonly used packages and tools. These downloadable files require
-little configuration, work on almost all setups, and provide all the
-commonly used scientific Python tools.
+## Environment-based workflows
 
-[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
-and Linux, provides over 1,500 Python packages, and is used by over 15
-million people. Anaconda is best suited to beginning users; it provides
-a large collection of libraries all in one.
+In environment-based workflows, you install packages into an environment, which you
+can activate and deactivate.
 
-For more advanced users who will need to install or upgrade regularly,
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) is a more
-suitable way to install the `conda` (and `mamba`, a faster `conda` alternative)
-package manager.
+If you have installed Python, you can create a virtual environment with a tool such
+as `venv` - see [the tutorial in the Python Packaging User Guide][venv-guide].
+You can then install [SciPy from PyPI], the Python Package Index, with [`pip`]:
 
-Other options include:
+    python -m pip install scipy
 
-- [WinPython](https://winpython.github.io): Another free distribution
-  including scientific packages and the Spyder IDE; Windows only, but
-  more actively maintained and supports the latest Python 3 versions.
-- [Pyzo](https://pyzo.org): A free distribution based on Anaconda
-  and the IEP interactive development environment; Supports Linux,
-  Windows, and Mac.
+[venv-guide]: https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments
+[SciPy from PyPI]: https://pypi.org/project/scipy/
+[`pip`]: https://pip.pypa.io/en/stable/getting-started/
 
-<a name="pip-install"></a>
+Alternatively, you can install [SciPy from conda-forge] using a [Conda]-based
+environment manager. You can install Python itself, as well as packages in
+languages other than Python, as Conda packages.
 
-# Installing with Pip
+[Miniforge] is the recommended way to install `conda` and [`mamba`],
+two Conda-based environment managers. You can then install SciPy as follows:
 
-You can install SciPy from PyPI with `pip`:
+    conda install scipy # or
+    mamba install scipy
 
-    python -m pip install scipy
+[SciPy from conda-forge]: https://anaconda.org/conda-forge/scipy
+[Miniforge]: https://conda-forge.org/download/
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+[`mamba`]: https://mamba.readthedocs.io/en/latest/
+
+<a name="project-based"></a>
+
+## Project-based workflows
+
+In project-based workflows, a project is a directory containing a manifest
```

Comment:
So, who are we targeting here? I can think of two groups:

(1) User who got hold of some random package, now wants to get that package ready for use. Is it our responsibility to provide package development instructions? Probably not.

(2) A user who is starting out with their own application , and wants to add a mechanism for setting up the exact right environment, including SciPy. This is a more reasonable group to target—although again I'm not sure it's the right place. Maybe simply point to `uv` and `pixi` website documentation?

Either way, these users are advanced, so they won't be arriving at the SciPy docs, trying to figure out how to add SciPy to uv/pixi envs.

Reply:
I would disagree that people using `uv` / `pixi` are always advanced, we get _a lot_ of beginner questions, i.e., "this is my first time using a Python project".

Anyway, if the installation is ever more complicated than `uv add scipy` I'm happy to add a dedicated integration page to the uv documentation.

Reply:
My question is more: "what is the path that takes someone to the SciPy installation page to ask about uv" and then "did they land in the right spot, or should they be sent somewhere else". I.e., if they're asking the right question, relevant to SciPy, we should absolutely answer it. If there's a more general resource we can point to, best to do that (community wide).

Reply:
@lucascolley I think if you want to describe, from an absolute beginner's perspective, how to get an environment set up using uv, that's fine too. What was there felt like too high level for a newcomer to start with. You'd have to explain which directory they should be in, how to activate the environment, how often they need to do that, how to install packages (per your snippet above), etc. And then it really feels more like an alternative env manager, rather than "project based". People who want to set up projects *should* be able to find good guides out there (pyopensci?).

Reply:
> You'd have to explain which directory they should be in, how to activate the environment, how often they need to do that, how to install packages (per your snippet above), etc. And then it really feels more like an alternative env manager, rather than "project based".

Okay, sounds good. I guess that would mean we can avoid repeating similar instructions for `pip` and `conda`? If so, is the recommendation of `uv`/`pixi` for new users justified?

Reply:
isn't it reasonable to assume that `uv` and `pixie` users already know how to install `scipy` once they've seen that `pip install scipy` is an option?

Reply:
> isn't it reasonable to assume that uv and pixie users already know how to install scipy

Yeah, so I think `uv add scipy` in itself wasn't helpful. We should instead be targeting people who are not experienced with either `uv`/`pixi` or `pip`/`conda` yet.